src/aes_decrypter.cpp

```diff
@@ -9,7 +9,9 @@
 #include "aes_decrypter.h"
 #include "utils/log.h"
 #include <bento4/Ap4StreamCipher.h>
+#ifndef INPUTSTREAM_TEST_BUILD
 #include <kodi/Filesystem.h>
+#endif
```

Comment:
i agree your change but please comment the code
```suggestion
// #ifndef INPUTSTREAM_TEST_BUILD
// #include <kodi/Filesystem.h>
// #endif
```
this is part of `RenewLicense` method that has been temporary comment out for a future full removal
has been kept only to see if some user complain about this old undocumented feature removal